src/transaction.ts

```diff
@@ -681,7 +680,7 @@ export class Snapshot extends EventEmitter {
    *   // ]
    * });
    *
-   * @example <caption>The SQL query string can contain parameter placeholders.
```

Comment:
Is it a foregone conclusion that it *will* contain placeholders? The meaning of this text was changed. @callmehiphop

Reply:
They are definitely optional, this should probably be reverted.